snaps/tutorials/gas-estimation.md

```diff
@@ -46,19 +53,18 @@ Edit the section under `npm` to change the `iconPath` to your new icon:
       "packageName": "snap",
       "registry": "https://registry.npmjs.org/"
    }
+}
 ```
 
-Then, place
-[this file](https://raw.githubusercontent.com/Montoya/gas-fee-snap/main/packages/snap/images/gas.svg)
-into the `/packages/snap/images` folder.
+Next, download [this free `gas.svg` icon file](https://raw.githubusercontent.com/Montoya/gas-fee-snap/main/packages/snap/images/gas.svg) into the `/packages/snap/images` folder.
```

Comment:
```suggestion
Next, download [this `gas.svg` icon file](https://raw.githubusercontent.com/Montoya/gas-fee-snap/main/packages/snap/images/gas.svg) into the `/packages/snap/images` folder.
```
Since free is mentioned in the next sentence.